modules/unsupported/arcgis-rest/README.rst

```diff
@@ -19,5 +19,5 @@ System tests
 
 Make sure to have all the necessary servwer certificates in a truststore.
```

Comment:
Another typo on this line: servwer -> server.

Reply:
Corrected.